### PR TITLE
Run tests on PHPUnit 9

### PR DESCRIPTION
**Pull request by SimonFrings**
For further details concerning this pull request look into [graphp/graphviz #46](https://github.com/graphp/graphviz/pull/46).

### CODE REVIEW

File: tests/TestCase.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            }
+            if ($exceptionCode !== null) {
+                $this->expectExceptionCode($exceptionCode);
+            }
+        } else {
+            // legacy PHPUnit 4
+            parent::setExpectedException($exception, $exceptionMessage, $exceptionCode);
+        }
+    }
+}
```
Comment (clue) [inline tests/TestCase.php]: nit: missing EOL

File: tests/TestCase.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+use PHPUnit\Framework\TestCase as BaseTestCase;
+
+class TestCase extends BaseTestCase
+{
+    protected function expectCallableOnce()
+    {
+        $mock = $this->createCallableMock();
+
+        $mock
```
Comment (clue) [inline tests/TestCase.php]: nit: inconsistent indentation, see following methods.

**Review by clue — CHANGES_REQUESTED** [review]
@SimonFrings Thanks for looking into this! Only some minor remarks below

File: tests/TestCase.php (hunk 1 above)
Comment (SimonFrings) [reply tests/TestCase.php]: Done :+1:

File: tests/TestCase.php (hunk 2 above)
Comment (SimonFrings) [reply tests/TestCase.php]: Everything should be fine now :+1:

**Review by clue — APPROVED** [review]
@SimonFrings Thanks for the update, changes LGTM! :shipit:

**Pull request merged**